web/repos.jspf

```diff
@@ -263,8 +264,13 @@ Copyright (c) 2015, 2016, Oracle and/or its affiliates. All rights reserved.
                                 if (branch == null) {
                                     branch = "N/A";
                                 }
+                                String currentVersion = ri.getCurrentVersion();
```

Comment:
There are two places in `repos.jspf` where a line about repository is being printed to the output. So this should be handled in both cases.